src/plexil-adapter/OwInterface.cpp

```diff
@@ -478,96 +478,83 @@ OwInterface::OwInterface ()
 {
 }
 
-OwInterface::~OwInterface ()
-{
-  if (m_genericNodeHandle) delete m_genericNodeHandle;
-  if (m_antennaTiltPublisher) delete m_antennaTiltPublisher;
-  if (m_leftImageTriggerPublisher) delete m_leftImageTriggerPublisher;
-  if (m_antennaTiltSubscriber) delete m_antennaTiltSubscriber;
-  if (m_antennaPanSubscriber) delete m_antennaPanSubscriber;
-  if (m_jointStatesSubscriber) delete m_jointStatesSubscriber;
-  if (m_cameraSubscriber) delete m_cameraSubscriber;
-  if (m_socSubscriber) delete m_socSubscriber;
-  if (m_rulSubscriber) delete m_rulSubscriber;
-  if (m_batteryTempSubscriber) delete m_batteryTempSubscriber;
-  if (m_instance) delete m_instance;
-}
+OwInterface::~OwInterface (){}
```

Comment:
We can get rid of the constructor, since it does not serve a purpose (by declaring it explicitly)
```suggestion
```

Reply:
Good (and minor) point, though I like classes to be explicit about their intentions regarding the compiler-generated functions, so my preference would be to either define it (empty in this case), or declare it (`= default` in this case).

Reply:
Did you see the link I provided on "the rule of the three"? You only need to go this route if you define any of the three (destructor, copy constructor, assignment constructor). If you did not declare any one of them then the compiler won't generate the others. It is a minor comment, it just helps keep the code concise.